app/battle/preparation.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+from app.classes.class_knights import Knights
+
+
+def protection_calc(knight: Knights, armour: list) -> None:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]

Reply:
Done